src/main/java/org/cp/confisual/plantuml/Visualiser.java

```diff
@@ -0,0 +1,38 @@
+package org.cp.confisual.plantuml;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import net.sourceforge.plantuml.SourceStringReader;
+import net.sourceforge.plantuml.security.SFile;
+import org.cp.confisual.nevisauth.Domain;
+import org.cp.confisual.nevisauth.NevisAuthConfigVisitor;
+
+public class Visualiser {
+  public static String IMG_EXTENSION = ".PNG";
+
+  private static SourceStringReader reader;
+
+  public static void visualise(NevisAuthConfigVisitor visitor, Domain domain, Path destinationFolder) throws VisualizerException {
```

Comment:
This method should only accept nevisauth's config file path. It's job is to parse the file, build the UML then output the image